src/renderer/views/app/components/NavigationButtons/style.ts

```diff
@@ -1,10 +1,6 @@
-import styled, { css } from 'styled-components';
-import { platform } from 'os';
+import styled from 'styled-components';
 
 export const StyledContainer = styled.div`
   display: flex;
   -webkit-app-region: no-drag;
-  ${({ isFullscreen }: { isFullscreen: boolean }) => css`

```

Comment:
The `isFullscreen` prop should be removed from there:
https://github.com/wexond/desktop/blob/25ac0ae8eb0c7ef02a9381a9e0f7cd079726527a/src/renderer/views/app/components/NavigationButtons/index.tsx#L40

Reply:
Just pushed commit 3c93bc5808e7fb04521a41a1485c73d88927469e which resolves this issue